mentees/README.md

```diff
@@ -0,0 +1,264 @@
+6. ***Feedback and evaluation:*** 
+Look for programs that provide regular feedback and evaluations to help you track your progress and adjust your goals as needed.
+
+As a final note, reach out to program graduates via the channels listed in the **'Support Networks'** section for guidance; or review related blogs, articles or videos to help inform your decision. 
+
+To maximize the benefits of a mentorship program, it is important to choose one that aligns with your personal goals and objectives. Each program has its own unique strengths and benefits, so by applying and learning more about the experience as it progresses, you can ensure that you select the best program for your needs.
+
+
+## **Remuneration**
+
+> **How do people find paid work through Open Source contribution?**
+
+*(Points/questions for consideration as below)*
+
+**What are the job roles and job descriptions that are available in the industry?**
+
+**How long does it generally take for someone to start earning?**
+
+**What can I do to increase my employability/attractiveness to potential employers?**
+
+## **Skills & Experience**
+
+As a contributor and potential mentee, it's important to have a diverse technical and professional skillset in order to make the most impact. CNCF projects rely on a wide range of skills to thrive, including software development, design, project management, documentation, testing, and more.
+
+***Choosing a project to contribute to***
+
+Selecting a project according to: *(paragraph needed)*
+* personal interests and preferences
+* existing or desired skills such as specific programming languages
+* reviewing the [CNCF Landscape](https://landscape.cncf.io/) to gain an understanding of different projects and researching to determine the best match
+* reviewing graduate profiles and communications such as blogs, Tweets and **Success Stories** to learn more about individual experiences. 
+
+
+***Developing as a professional alongside your mentor***
+Your technical experience will be critical to your work during your mentorship. Professional excellence however can only be achieved by balancing that knowledge with 
+
+
+
+Whether you are a developer, designer, or anything in between, there is a place for you in mentoring programs on offer, and your unique skills and perspectives will be valuable assets to any project you choose to contribute to.
+
+
+* Provide general overview of skills common in recent LFX projects e.g. Go (provide link to skills map or similar resource from previous intake as indication?)
+* Dependent on individual projects, subject to change over time
+* Emphasise soft skills such as communication; critical regardless of technical expertise required
+
+## **Success Stories**
+
+*(Refer to existing repo content; additions as required)*
+
+---
+
+
+# **FAQs**
+
+ ### **I’m new to Open Source and CNCF. Where should I start?**
```

Comment:
How about we link prominently in the answer to the new content at https://contribute.cncf.io/contributors/getting-started/? I feel like there's a lot of overlap